server/sonar-server/src/main/java/org/sonar/server/ws/WsParameterBuilder.java

```diff
@@ -45,6 +45,13 @@ private WsParameterBuilder() {
       .setPossibleValues(getRootQualifiers(context.getResourceTypes()));
   }
 
+  public static WebService.NewParam createRootQualifiersParameter(WebService.NewAction action, QualifierParameterContext context) {
+    return action.createParam(PARAM_QUALIFIERS)
+      .setDescription("Comma-separated list of component qualifiers. Filter the results with the specified qualifiers. " +
+        "Possible values are:" + buildRootQualifiersDescription(context))
+      .setPossibleValues(getRootQualifiers(context.getResourceTypes()));
```

Comment:
As api/projects/search is using a default value 'TRK' for qualifiers, why not using the same here ?
For consistency I think it would be better

Reply:
it's done in the BulkApplyTemplate action, and this class is used along the whole permission domain, so I don't want to change that

Reply:
Then I would propose to add the default value in BulkApplyTemplateAction